nav2_map_server/test/component/test_occ_grid_node.cpp

```diff
@@ -16,17 +16,19 @@
 #include <rclcpp/rclcpp.hpp>
 #include <memory>
 
-#include "nav2_map_server/occ_grid_loader.hpp"
 #include "test_constants/test_constants.h"
+#include "nav2_map_server/occ_grid_loader.hpp"
+#include "nav2_util/lifecycle_service_client.hpp"
 
-// TODO(bpwilcox): test publisher
```

Comment:
This reminds me that we should add a publisher test case!

Reply:
@bpwilcox - please file an issue so we don't forget